### PR TITLE
FE-1348 - Resolve high severity Dependabot alerts

### PR DESCRIPTION
**Pull request by nicklemmon**
[FE-1348](https://sparkpost.atlassian.net/browse/FE-1348)

### What Changed

- Uninstalls unused and unmaintained `fork-ts-checker-webpack-plugin-alt`
- Updates `terser-webpack-plugin` to use version with updated dependency
- Uninstalls `mdx-loader` which depended on a package with a security vulnerability
- Upgrades `axios` to version 0.21.1
- Removes Storybook from the repo - fixing a vulnerability would have required a major overhaul - and we aren't really using it! Good to remove dependencies we aren't using
- Manually installs `raw-loader` - this must have been a transitive dependency of one of the above previously

### How To Test

- Ensure the app continues to build
- Ensure tests in CI continue to pass
- Manually test core flows and verify they still work as expected

### To Do

- [ ] Incorporate feedback

### CODE REVIEW

**Comment by nicklemmon** [comment]
@shrirupa Just rebased this one - let me know if there are questions that come up.

File: src/hooks/useSparkPostQuery/useSparkPostQuery.mdx [hunk 1]
```diff
@@ -1,6 +1,4 @@
-import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
-
-<Meta title="Hooks|useSparkPostQuery" />
+# useSparkPostQuery
```
Comment (shrirupa) [inline src/hooks/useSparkPostQuery/useSparkPostQuery.mdx:1]: I wonder why rest of the files don't look like the change of location diff, like this one

File: src/reducers/currentUser.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ export default (state = initialState, { type, payload, meta }) => {
         cookie_consent: !!payload.cookie_consent,
         // TODO: Remove when OG theme is removed
         // Just defaults the app state to having Hibana enabled when using the env variable
-        // Run the app locally with `npm run start-app-hibana`
+        // Run the app locally with `npm run start-hibana`
```
Comment (shrirupa) [inline src/reducers/currentUser.js:27]: 👍  good catch!

**Review by shrirupa — APPROVED** [review]
Looks good to me!

**Pull request merged**